frontend/src/scenes/experiments/ExperimentMetricForm.tsx

```diff
@@ -198,27 +228,46 @@ export function ExperimentMetricForm({
                 <LemonLabel
                     className="mb-1"
                     info={
-                        <>
-                            The preview uses data from the past 14 days to show how the metric will appear.
-                            <br />
-                            For funnel metrics, we simulate experiment exposure by inserting a page-view event at the
-                            start of the funnel. In the experiment evaluation, this will be replaced by the actual
-                            experiment-exposure event.
-                        </>
+                        <div className="flex flex-col gap-2">
+                            <div>This shows recent activity for your selected metric over the past 2 weeks.</div>
+                            <div>
+                                It's a quick health check to confirm your tracking is working correctly and that you can
+                                expect meaningful results once your experiment starts running.
```

Comment:
```suggestion
                                It's a quick health check to ensure your tracking is working properly, so that you'll
                                receive accurate results when your experiment starts.
```